waspc/src/Wasp/Psl/Ast/Model.hs

```diff
@@ -1,64 +1,21 @@
-{-# LANGUAGE DeriveDataTypeable #-}
-
 module Wasp.Psl.Ast.Model
   ( Model (..),
-    Body (..),
```

Comment:
I know you moved these to `Model.Common`, but now you have a lot of imports that have to reference `Model.Common` instead of `Model` which is a bit weird. What you could do instead is re-export this stuff from here again! So it behaves same like before, but it will now leave in Common.